genomic_address_service/call.py

```diff
@@ -105,22 +105,20 @@ def run_call(config):
     write_threshold_map(threshold_map, os.path.join(outdir, "thresholds.json"))
 
     obj = assign(dist_file,membership_file,threshold_map,linkage_method,address_col,sample_col,batch_size)
-
+    print(obj.memberships_dict)
```

Comment:
Could you either include some text describing what this data is (e.g., "The memberships_dict="), or remove this print statement if it is no longer needed? Some with the `threshold_map` print statement below.